### PR TITLE
0.12 Updates

### PR DESCRIPTION
**Pull request by safareli**
Please enable travis to see if everything works fine

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -6,23 +6,22 @@
...
+    "purescript-effect": "^2.0.0",
+    "purescript-quickcheck": "^5.0.0",
+    "purescript-maybe": "^4.0.0",
+    "purescript-arrays": "^5.0.0",
+    "purescript-leibniz": "^5.0.0",
+    "purescript-foldable-traversable": "^4.0.0",
+    "purescript-unsafe-coerce": "^4.0.0",
+    "purescript-newtype": "^3.0.0",
+    "purescript-integers": "^4.0.0",
+    "purescript-quotient": "safareli/purescript-quotient#compiler/0.12",
```
Comment (safareli) [inline bower.json]: https://github.com/rightfold/purescript-quotient/pull/2 should me merged and released first.

**Comment by safareli** [comment]
I'm getting:
```

100/100 test(s) passed.
reverse
100/100 test(s) passed.
foldl
100/100 test(s) passed.
foldr
100/100 test(s) passed.
fromString
100/100 test(s) passed.
0/100 test(s) passed.
/Users/safareli/dev/foam/purescript-bytestrings/output/Effect.Exception/foreign.js:29
    throw e;
    ^

Error: Test 1 (seed 462831653) failed:
(Just (pack [])) /= Nothing
    at Object.exports.error (/Users/safareli/dev/foam/purescript-bytestrings/output/Effect.Exception/foreign.js:8:10)
    at /Users/safareli/dev/foam/purescript-bytestrings/output/Test.QuickCheck/index.js:139:81
    at /Users/safareli/dev/foam/purescript-bytestrings/output/Data.Foldable/index.js:95:76
    at /Users/safareli/dev/foam/purescript-bytestrings/output/Data.Foldable/index.js:292:24
    at /Users/safareli/dev/foam/purescript-bytestrings/output/Data.Foldable/index.js:417:46
    at /Users/safareli/dev/foam/purescript-bytestrings/output/Data.Function/index.js:19:24
    at __do (/Users/safareli/dev/foam/purescript-bytestrings/output/Test.QuickCheck/index.js:138:122)
    at __do (/Users/safareli/dev/foam/purescript-bytestrings/output/Test.QuickCheck/index.js:169:68)
    at Object.__do [as main] (/Users/safareli/dev/foam/purescript-bytestrings/output/Test.Main/index.js:162:294)
    at Object.<anonymous> (/private/var/folders/61/zph7lsls1114q0727nst6yt80000gn/T/pulp-test1181021-63652-braywu.3zr3.js:1:84)
* ERROR: Subcommand terminated with exit code 1
```

**Comment by rightfold** [comment]
I have enabled Travis. Could you do an amend and force push to trigger it?

**Comment by safareli** [comment]
can this be merged and released as it's passing now?

**Pull request merged**